cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.5
+
+- The Billing Center dimension was added to show the corresponding results
```

Comment:
This one can be more similar to other updates in the file, you could put something like:
`added billing center dimension to show the corresponding results`.

Reply:
Comment Addressed. Thank you